app/src/main/java/space/taran/arknavigator/mvp/model/repo/nodes/FolderNodesRepo.kt

```diff
@@ -0,0 +1,11 @@
+package space.taran.arknavigator.mvp.model.repo.nodes
+
+import space.taran.arknavigator.mvp.presenter.adapter.folderstree.FolderNode
+
+class FolderNodesRepo(
+    private val nodesProvider: FolderNodesProvider
+) {
+    fun save(nodes: List<FolderNode>) = nodesProvider.save(nodes)
+
+    fun provide() = nodesProvider.provide()
+}
```

Comment:
As far as I understand the main idea is to have an application level cache for folder tree items. Not sure we need this.